websites/S/Scan/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "DubStepMad",
+      "id": "118810952530395136"
+    },
+    "url": "www.scan.co.uk",
+    "description": {
+      "en": "Official Scan computer parts, computing services and business."
+    },
+    "service": "Scan",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/14KOuEk.png",
+    "thumbnail": "https://i.imgur.com/egAfDfZ.png",
+    "color": "#ffffff",
+    "tags": [
```

Comment:
probs better to do their logo blue color instead of just full white

Reply:
Implemented in updated pull.